examples/topologies/simple-gateway/meta.yaml

```diff
@@ -0,0 +1,8 @@
+title: Simple Gateway
+description: |-
+  [Gateways][gateways] can be used to form super-clusters which propagate the
```

Comment:
Hm, I know its a simple example, but I think we should at least mention, that users should follow N/2 + 1 for the number of gateways and that each "cluster" should be an actual cluster in supercluster setup  (maybe except arbiter if that's the case) in anything beyond local dev/lab ends.

Maybe even connect 3 single-node gateways? :).
Not much more lines in example.

Reply:
Good point. I think I will distinguish a non-JS supercluster setup and a JS-enabled supercluster setup since the latter is more nuanced for this reason.